cdc/owner.go

```diff
@@ -489,10 +490,14 @@ func (o *ownerImpl) loadChangeFeedInfos(ctx context.Context) error {
 				continue
 			}
 
+			startTs := changefeed.StartTs
+			if checkpointTs > startTs {
```

Comment:
seems `detail.GetCheckpointTs() >= detail.StartTs` should always be true or it's unreasonble

Reply:
yep, change to use `detail.GetCheckpointTs()` directly